### PR TITLE
Add tooltip to save measurements input box

### LINKED ISSUE #120: Add tooltip to save measurements input box
Add a tooltip to the save measurement input box.  Features:

- allow multiple lines in text (such as measurementTooltips)
- Privacy policy link should open new browser window with our privacy policy; ideally it should be clean window with nothing extra (no header or footer) along the lines of:

`<a href="https://sizeme.com/privacy_clean.html"
   onclick="window.open(this.href,'targetWindow',
                                   'toolbar=no,
                                    location=no,
                                    status=no,
                                    menubar=no,
                                    scrollbars=yes,
                                    resizable=yes,
                                    width=600,
                                    height=400');
 return false;">Privacy Policy</a>`

But I don't know the usability of that on mobile.

### PR DESCRIPTION
**Pull request by tjjalava**
Fixes #120

### CODE REVIEW

File: src/i18n/ar.json [hunk 1]
```diff
@@ -166,7 +166,11 @@
 	"emailPlaceholder": "myname@example.com",
     "message": "حفظ قياساتك",
 	"save": "حسنا",
-    "signupPrompt": "حفظ بواسطة البريد الإلكتروني"
+    "signupPrompt": "حفظ بواسطة البريد الإلكتروني",
+    "tooltipBullet1": "!!Save your measurements so we will remember you next time!!!",
```
Comment (tjjalava) [inline src/i18n/ar.json]: @nomasi translations, por favor

File: src/i18n/fi.json [hunk 2]
```diff
@@ -175,7 +175,11 @@
     "measurementDisclaimerCollar": "Paitsi kaulus, joka on ympärysmitta kauluksen sisäpuolelta.",
     "measurementDisclaimerInside": "Nämä mitat on otettu <u>tuotteen sisältä</u>.",
     "tableTitle": "Tuotteen mitat",
-    "windowTitle": "Kokotaulukko tuotteelle"
+    "windowTitle": "Kokotaulukko tuotteelle",
+    "tooltipBullet1": "!!Save your measurements so we will remember you next time!!!",
```
Comment (tjjalava) [inline src/i18n/fi.json]: @nomasi and here

File: src/i18n/sv.json [hunk 3]
```diff
@@ -176,7 +176,11 @@
     "measurementDisclaimerCollar": "Med undantag av kragmåttet, som är uppmätt på kragens insida.",
     "measurementDisclaimerInside": "Dessa mått är uppmätta <u>på insidan av artikeln</u>.",
     "tableTitle": "Aktuella artikel mått",
-    "windowTitle": "Storleks Guide för"
+    "windowTitle": "Storleks Guide för",
+    "tooltipBullet1": "!!Save your measurements so we will remember you next time!!!",
```
Comment (tjjalava) [inline src/i18n/sv.json]: @nomasi and here

Comment (nomasi) [reply src/i18n/sv.json]: The Swedish translation isn't used, so let's fix this when we have a use-case.

**Pull request merged**